packages/eslint-plugin/src/types/rules.ts

```diff
@@ -4,9 +4,9 @@ type CodePath = Rule.CodePath;
 type CodePathSegment = Rule.CodePathSegment;
 
 interface TSRuleListener {
```

Comment:
The comment below reminds me - this should be called `Visitor`, not `Listener`

Reply:
They call it rule listener in the vanila eslint types, I think we should follow the convention. Let me know if you prefer we change.